Specs/DateTime/Spanish/DateTimePeriodParser.json

```diff
@@ -2891,5 +2856,55 @@
         "Length": 21
       }
     ]
+  },
```

Comment:
It seems the source of most issues was a translation confusion between "in" vs "within".
In English the current behaviour is:
"in 3 min" -> "3 min from now" -> datetime
"within 3 min" -> "from now until 3 min later" -> range
Which it seems you managed to correct or adapt the mistranslated specs into the right output.
Could you add a couple more spec cases to the appropriate Parsers and DateTimeModel spec files that closely match the 4 cases above? Some may already exist.

Reply:
added cases to Parsers and Model. 
The case "from now until 3 min later" maybe requires additional fixes because the parsed Timex is "PT0H" instead of "PT3M"

Reply:
Got it. Please add it as the next in the task stack.

Reply:
fixed